pkg/ebpftracer/tracer_decode.go

```diff
@@ -138,6 +143,13 @@ func (t *Tracer) decodeAndExportEvent(ctx context.Context, data []byte) (rerr er
 			},
 		}
 	case types.SchedProcessExecArgs:
+    fmt.Println("===>", eventCtx.MntID, eventCtx.HostPid, eventCtx.NodeHostPid, eventCtx.Pid, args.Pathname, args.Argv)
+		if eventCtx.Pid == 1 {
+			t.cfg.MountNamespacePIDBookkeeper.ForceAddToBucket(proc.NamespaceID(eventCtx.MntID), eventCtx.NodeHostPid)
```

Comment:
Nitpick: `MountNamespacePIDStore` could be a bit more clear. Bookkeeper made me think more what it does. But this just stores mout ns to pids mapping.